tools/dds/dds-sniffer/dds-sniffer.hpp

```diff
@@ -28,12 +30,23 @@ class Sniffer : public eprosima::fastdds::dds::DomainParticipantListener
     std::map< eprosima::fastdds::dds::DataReader *, eprosima::fastrtps::types::DynamicType_ptr > _readers;
     std::map< eprosima::fastdds::dds::DataReader *, eprosima::fastrtps::types::DynamicData_ptr > _datas;
 
+    std::map< eprosima::fastrtps::rtps::GUID_t, eprosima::fastrtps::string_255 > _discoveredParticipants;
+    struct ReadersWriters
```

Comment:
I think this is something along the lines of a `topic information`? So `topic_info`?